app/src/components/layout/Header/Header.styled.ts

```diff
@@ -12,17 +12,15 @@ export const HeaderComponentsContainer = styled.div<{
   isMobile: boolean;
 }>`
   display: flex;
-  flex-direction: row;
   justify-content: space-between;
   width: 100%;
   max-width: ${pxToRem(1400)};
   width: ${({ isMobile }) => (isMobile ? '100%' : '98%')};
   margin: ${({ isMobile }) => (isMobile ? '0' : '0 auto')};
-
-  padding: 1.75rem 1.7rem 1.7rem 2.17rem;
+  padding-right: 0.8rem;
 
   @media (min-width: ${breakpoints.lg}) {
-    padding: 3.5rem 2rem 1.75rem 2rem;
+    padding-right: 0;
   }
```

Comment:
Looks like you're overriding left, top and bottom padding.  Is that intentional?